package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
+    "export": "next build && next export",
```

Comment:
Do we need to leave this in? It's unlikely that anyone will ever have to manually export the website, and next export should complain that you need to run next build first :D